src/components/templates/box/index.d.ts

```diff
@@ -33,3 +33,9 @@ export type BoxProps<T extends React.ElementType = any> = CombinedStyledProps<T>
   as?: React.ElementType
   sx?: CombinedStyledProps<T>
 }
+
+declare const Box: React.FC<BoxProps & JSX.IntrinsicElements["div"]>
+
```

Comment:
We don't need JSX.IntrinsicElements["div"]. Replace it with React.FC<BoxProps<"div">>. Box props is extending `React.ElementType ` which uses JSX.IntrinsicElements under the hood.